### DISCUSSION TITLE
Fast Reaction Rate

### DISCUSSION THREAD

**Discussion by PavelMuravskii** [GeoChemFoam/GeoChemFoam#52] [Q&A] [answered]
Hello Dear GeoChemFoam developers and community!

I found this repository not so long ago. Maybe someone checked how physical the solutions of this repository (DBSFoam or ALEFoam) are when the concentration rises or just the reaction constants increase?(The question about increasing the reaction rate --- like example when the flow rate is approximately equal to the reaction rate). And if it didn't work out, who solved this problem and how? Or can this CeochemFoam be used to calculate faster reactions?

It's just that, as I understand from the article, a Quasistic assumption is used here - which assumes that the reaction of the mineral dissolution is much slower than the flow velocity. For this reason, this question has appeared

**Comment by GeoChemFoam** [comment, marked as answer]
Hello,

You’re really going to have to be more specific. GeoChemFoam has 16 solvers and more than one handles reaction.

The iVOS solver has been benchmarked with reaction rates (Ki #) 100x higher than flowrate (Pe #). See https://www.nature.com/articles/s41598-023-37725-6 and  https://www.frontiersin.org/journals/earth-science/articles/10.3389/feart.2022.917931/full for details.

Best wishes,

Hannah

From: PavelMuravskii ***@***.***>
Date: Saturday, 19 October 2024 at 13:10
To: GeoChemFoam/GeoChemFoam ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GeoChemFoam/GeoChemFoam] Fast Reaction Rate (Discussion #52)
****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Hello everyone, I found this repository not so long ago. Maybe someone checked how physical the solutions of this repository are when the concentration rises or just the reaction constants increase?(The question about increasing the reaction rate --- like example when the flow rate is approximately equal to the reaction rate). And if it didn't work out, who solved this problem and how?

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/52>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWDOAFR52E5V6CY54M4UDQDZ4JD4TAVCNFSM6AAAAABQHNWC5CVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGM2DANZRGA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

  1.  Heriot-Watt University, a Scottish charity registered under number SC000278
  2.  Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.

**Reply by PavelMuravskii** [reply]
Thanks for your answer, I will deal with this topic. THANKS
